ci/utils.py

```diff
@@ -101,3 +102,9 @@ def get_ranged_data_from_timeseries(timeseries, dt, range_type='week'):
     end = get_timestamp(boundries.end)
     sent_data = transform_timeseries_data(timeseries, start, end)
     return right_pad_list(sent_data, length=padding, value=0)
+
+
+def extract_query_params(url_str):
+    if url_str is None:
```

Comment:
Would it not be better here to ensure that it's a string type, rather than just `None`?